Sources/Client/Views/Settings/UpdateView.swift

```diff
@@ -13,6 +13,19 @@ enum UpdateError: LocalizedError {
   case failedToGetDownloadURLFromGitHubReleases
   case failedToGetBranches(Error)
   case failedToGetGitHubAPIResponse(Error)
+  
+  var errorDescription: String? {
+    switch self {
+    case .failedToGetDownloadURL:
+      return "Failed to get download URL"
```

Comment:
Please end all errors with a full stop for consistency. It's my mistake if I've accidentally forgotten a full stop in any of the documentation comments (which I have done plenty of times 😅)